Adyen/UI/Form/Items/Value Pickers/Abstract/BaseFormPickerItemView.swift

```diff
@@ -30,40 +39,54 @@ open class BaseFormPickerItemView<T: CustomStringConvertible & Equatable>: FormV
         super.init(item: item)
         initialize()
         select(value: item.value)
+        observe(item.$selectableValues) { [weak self] change in
+            guard let self = self else { return }
+            self.inputControl.showChevron = change.count > 1
+            self.pickerView.reloadAllComponents()
+            change.first.map(self.select)
```

Comment:
maybe extract this in to a separate function, and then you don't need to strongify self?

Reply:
Tried it but the `change` param is too long to name as a param :)
`Observable<[BasePickerElement<ElementType>]>`